docs/spark/how-to-guides/debug.md

```diff
@@ -30,13 +30,42 @@ When you run the command, you see the following output:
 ***********************************************************************
 ```
 
-In this debug mode, `DotnetRunner` does not launch the .NET application, but it waits for it to connect. Leave this command prompt window open.
+In this debug mode, `DotnetRunner` does not launch the .NET application but instead waits for it to connect. Leave this command prompt window open.
```

Comment:
What are we waiting for the .NET application to connect to? Would that be worth mentioning that (i.e. "...waits for it to connect to a C# debugger.") explicitly here?

Reply:
Once the debugger is started on the scala side, it waits for user to start the .NET application, that's what 'it' means here. If it's not easily understandable we could make it like -

In this debug mode, `DotnetRunner` does not launch the .NET application but instead waits for it to connect. Leave this command prompt window open, and start your .NET application through C# debugger to debug your application.

Reply:
I think your suggestion here sounds great! Or, perhaps instead of still using "it" in that first sentence, you could add the clarification you had provided me (which I thought was nicely clear), i.e. "In this debug mode, DotnetRunner does not launch the .NET application but instead waits for you to start the .NET app. Leave this command..."

Reply:
Perfect, makes sense, will make the change. Thanks!